src/drivers/imx/esai.c

```diff
@@ -242,6 +249,12 @@ static inline int esai_set_config(struct dai *dai,
 			ESAI_xFCR_xWA(24) | ESAI_xFCR_xFWM(96)
 			| ESAI_xFCR_TE(1) | ESAI_xFCR_TIEN);
 
+	dai_update_bits(dai, REG_ESAI_RFCR,
+			ESAI_xFCR_xFR_MASK | ESAI_xFCR_xWA_MASK
+			| ESAI_xFCR_xFWM_MASK | ESAI_xFCR_RE_MASK,
+			ESAI_xFCR_xWA(24) | ESAI_xFCR_xFWM(96)
```

Comment:
Is it possible to leave | at the end of the line, so that the beginning of the lines are easier to read?